torch/nn/functional.py

```diff
@@ -3639,9 +3639,11 @@ def multi_head_attention_forward(query,                           # type: Tensor
         - value: :math:`(S, N, E)` where S is the source sequence length, N is the batch size, E is
           the embedding dimension.
         - key_padding_mask: :math:`(N, S)`, ByteTensor, where N is the batch size, S is the source sequence length.
+          ``1`` or ``True`` is filled with ``-inf`` while ``0`` or ``False`` is filled with ``0.0``.
```

Comment:
This is more confusing I think than to say "If float 1 and 0 mean XYZ. If bool True and False mean ABC".